doc/EnvironmentVariables.md

```diff
@@ -84,7 +84,10 @@
-| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                         |
-| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                       |
\ No newline at end of file
+| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                            |
+| MqttBroker__MaxQueuedMessages           | Maximum number of messages that can be in the outgoing queue for a single session                                                                  | int32                                                                 | 1000                          |
+| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                  |
+| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                      |
+| MqttBridge__StorageMaxDiskUsage         | If `UsePersistentStorage` is set, defines maximum size (in bytes) of the MQTT Bridge storage. This storage is separate from edgehub messages / twins storage.   | int64                                                    | 33554432 (32mb)               |
+| MqttBridge__StorageFlushOptions         | If `UsePersistentStorage` is set, defines MQTT Bridge storage flush options.                                                                                    | AfterEachWrite, Off                                      | AfterEachWrite                |
+| MqttBridge__StorageMaxMemoryMessages    | If `UsePersistentStorage` is not set, defines max in-memory queued messages in the MQTT Bridge.                                                                 | int32 (arm), int64 (amd64)                               | 1000                          |
```

Comment:
Actually thinking about this a bit more - one use case we have seen is limiting the memory used by these messages when using in memory store.

Reply:
Does it make sense to make the max usage env vars storage type agnostic (in-memory or disk)?

Reply:
@varunpuranik, it does make sense, but the issue is that those two vars are different units. Disk Usage is in bytes, where in-memory usage is in number of messages. This is how in-memory store was implemented. We can change that, but I'd rather do it in a separate PR.